app/templates/settings.html

```diff
@@ -130,44 +126,22 @@ <h1 style="text-align: center; padding-top: 1rem;">Change Password</h1>
                     <li class="large padding">
                         <fieldset class="settings-button center">
                             <div>
-                                {{ emailForm.submit(class_="save") }}
+                                {{ pwForm.submit(class_="save") }}
                             </div>
                         </fieldset>
                     </li>
                 </ul>
             </form>
 
-            <h1 style="text-align: center; padding-top: 1rem; width: 100vw; position: relative; margin: 0 auto;">Change Email Address</h1>
-            <form action="" method="post">
+            <h1 style="text-align: center; padding-top: 3rem; width: 100vw; position: relative; margin: 0 auto;">Change Email Address</h1>
+            <form action="{{ url_for('main.change_email') }}" method="post">
                 {{ emailForm.hidden_tag() }}
```

Comment:
```suggestion
```
Doesn't exist anymore, should be removed, together with the ending tag `</form>`.